### ISSUE TITLE
Binary Precision/Recall/F1ScoreMetric use unseeded random.choice() for pos_label, so identical inputs score anywhere from 0.0 to 0.889 across repeated calls

### ISSUE THREAD

**Issue by shaurya416** [argilla-io/argilla#5873]
`argilla-v1/src/argilla_v1/client/feedback/metrics/annotator_metrics.py:312` (`PrecisionMetric._compute`), the identical pattern repeats at `:333` (`RecallMetric._compute`) and `:354` (`F1ScoreMetric._compute`):

```python
@requires_dependencies("scikit-learn")
def _compute(self, responses, suggestions):
    from sklearn.metrics import f1_score

    if is_multiclass(responses) or is_multiclass(suggestions):
        kwargs = {"average": "macro"}
    else:
        kwargs = {"average": "binary", "pos_label": random.choice(np.unique(responses))}
    return f1_score(responses, suggestions, **kwargs)
```

For any binary `LabelQuestion`/`RatingQuestion`, which class counts as "positive" for `sklearn`'s `average="binary"` mode is chosen by an **unseeded `random.choice()`** on every single call to `compute()`. Because precision/recall/F1 under `average="binary"` are computed only against the chosen positive class, flipping `pos_label` between the majority and minority class changes the reported score, not just its sign convention.

## What happens (reproduced on the real module)

This affects `PrecisionMetric`, `RecallMetric` and `F1ScoreMetric` when the data for a `LabelQuestion` or `RatingQuestion` has exactly two distinct values (`is_multiclass()` is false). `AccuracyMetric` and the macro-averaged multiclass path are unaffected.

The file lives in `argilla-v1` (the 1.x SDK), which per its own README has stopped feature development but still receives bug fixes.

A minimal fix is to replace `random.choice(np.unique(responses))` with a deterministic sorted pick, e.g. `np.unique(responses)[-1]`, mirroring the existing sorted-label precedent at `MultiLabelMetrics._pre_process`. Making `pos_label` an explicit, documented parameter would be the more correct long-term fix, since which class is "positive" is a modeling decision, not something a library should decide by coin flip.

Happy to open the PR.
